Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -375,6 +398,21 @@ void Close()
 
 	SAFE_RELEASE(context);
 	ULONG references = device->Release();
+
+#if defined(_DEBUG) || defined(DEBUGFAST)
+	if (debug)
+	{
+		--references; // we didn't release the debug interface yet
```

Comment:
Why didn't we? Where is it released instead? What does `SAFE_RELEASE(debug)` do, since apparently it's not the debug interface? Why is references decremented instead of just moving this block in front of the actual `device->Release` call?

This comment needs some elaboration, because things are still confusing.